tests/containers/helm.pm

```diff
@@ -14,14 +14,37 @@ use version_utils qw(is_sle);
 use registration qw(add_suseconnect_product get_addon_fullname);
 use mmapi 'get_current_job_id';
 
+
+sub install_kubectl {
+    my $version = get_var('PUBLICCLOUD_KUBECTL_VERSION', 'v1.22.2');
+    assert_script_run("curl -LO https://dl.k8s.io/release/$version/bin/linux/amd64/kubectl");
+    assert_script_run("chmod a+x kubectl");
+    script_run("rm /usr/bin/kubectl");
+    assert_script_run("mv kubectl /usr/bin/kubectl");
+}
+
+sub install_aws_cli {
+    assert_script_run("pip3 install awscli");
```

Comment:
I would prefer doing this in the official way using Public Cloud Module
https://scc.suse.com/packages?name=SUSE%20Linux%20Enterprise%20Server&version=15.4&arch=x86_64&query=aws-cli


```
add_suseconnect_product(get_addon_fullname('pcm'));
zypper_call('in aws-cli');
```

Reply:
Done